client/landing/stepper/declarative-flow/free-post-setup.ts

```diff
@@ -0,0 +1,47 @@
+import { FREE_POST_SETUP_FLOW } from '@automattic/onboarding';
+import { translate } from 'i18n-calypso';
+import { useSiteSlug } from '../hooks/use-site-slug';
+import { recordSubmitStep } from './internals/analytics/record-submit-step';
+import FreePostSetup from './internals/steps-repository/free-post-setup';
+import { ProvidedDependencies } from './internals/types';
+import type { Flow } from './internals/types';
+
+export const freePostSetup: Flow = {
+	name: FREE_POST_SETUP_FLOW,
+	get title() {
+		return translate( 'Free' );
+	},
+	useSteps() {
+		return [ { slug: 'newsletterPostSetup', component: FreePostSetup } ];
```

Comment:
Is the slug supposed to be `freePostSetup` instead

Reply:
Definitely :). I had some unpushed commits locally. Just pushed. Should address this and some other issues.